examples/npm_lib/koboldNpmLib.js

```diff
@@ -0,0 +1,12 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+
+export async function koboldNpmLib() {
+    const wsProvider = new WsProvider('wss://rpc.polkadot.io');
+    const api = await ApiPromise.create({ provider: wsProvider });
+
+    return api.genesisHash.toHex();
```

Comment:
What's the type of `genesisHash`? You could potentially send raw binary from `Uint8Array` to Rust via `&[u8]` or `Vec<u8>` much more cheaply than via string (no Utf16 to Utf8 conversion necessary).

Reply:
it must return a String, since it can be passed to `.toHex()` which interpolates it `0x${string}`